File: R Codewars/7kyu/Vowel Count.R
# Description
# Return the number (count) of vowels in the given string.
# We will consider a, e, i, o, u as vowels for this Kata (but not y).
# The input string will only consist of lower case letters and/or spaces.

get_count <- function(input_str) {
  vowels <- c("a", "e", "i", "o", "u")
  count <- 0
  
  for (char in strsplit(input_str, "")[[1]]) {
    if (char %in% vowels) {
      count <- count + 1
    }
  }
  
  return(count)
}

File: C# Codewars/7kyu/Vowel Count.cs
/*
Description

Return the number (count) of vowels in the given string.

We will consider a, e, i, o, u as vowels for this Kata (but not y).

The input string will only consist of lower case letters and/or spaces.
*/

using System;
using System.Linq;

public static class Kata
{
  public static int GetVowelCount(string str)
  {
    return str.Count(i => "aeiou".Contains(i));
  }
}